resources/Hooks/useCountdown.js

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react'
+
+const useCountdown = (targetDate) => {
```

Comment:
a couple things to test here:

- basic case: does a date passed in get rendered correctly
- does a new date get rendered after 1 second elapses
- do the correct extensions get rendered for minute, hour, etc
- does "expired" get rendered in the correct scenarios
- what happens when we pass in a date in the past or an invalid date

Reply:
this article may have some good pointers:
https://kentcdodds.com/blog/how-to-test-custom-react-hooks